php/libraries/UserPermissions.class.inc

```diff
@@ -106,30 +106,63 @@ class UserPermissions
 
 
     /**
-     * Determines if the user has a permission
+     * Determines if the user has the permission to access a specific module
+     * or page.
      *
-     * @param string $code The permission code
+     * @param string $code     The permission code
+     * @param string ...$codes Optional additional permission codes
      *
      * @return bool
      */
-    function hasPermission($code)
+    function hasPermission($code, ...$codes): bool
+    {
+        /* True if $code is the only parameter passed and it is a valid
+         * permission code possessed by the user.
+         */
+        $hasSinglePermission = $this->permissionTrueAndValid($code)
```

Comment:
Well written, it could have been a bit easier with an if statement, but the logic is correct